File: output/Llama-2-100/avatar/Java/Python/codeforces_31_A.py
def codeforces_31_A ( n ):
    a = [ int ( i ) for i in range ( n + 1 ) ]
    for k in range ( n ):
        t = a [ k ]
        for i in range ( n - 1 ):
            for j in range ( n - 1 ):
                if i != j and t == ( a [ i ] + a [ j ] ):
                    return 1
                if t == ( a [ i ] + a [ j ] ):
                    return 1
    return 0

File: output/StarCoder-100/avatar/Python/Java/codeforces_31_A.java
import java.util.*;
public class codeforces_31_A {
    public static void main ( String [ ] args ) {
        Scanner sc = new Scanner ( System.in );
        int n = sc.nextInt ( );
        int [ ] arr = new int [ n ];
        for ( int i = 0 ; i < n ; i ++ ) {
            arr [ i ] = sc.nextInt ( );
        }
        for ( int i = 0 ; i < n ; i ++ ) {
            for ( int j = 0 ; j < n ; j ++ ) {
                if ( i == j ) {
                    continue;
                }
                for ( int k = 0 ; k < n ; k ++ ) {
                    if ( arr [ k ]!= arr [ j ] && arr [ k ]!= arr [ i ] && arr [ k ] == arr [ i ] + arr [ j ] ) {
                        System.out.println ( k + 1 + " " + j + 1 + " " + i + 1 );
                        return;
                    }
                }
            }
        }
        System.out.println ( - 1 );
    }
}

